src/lib.rs

```diff
@@ -596,3 +650,28 @@ impl<'a> LabelBuilder<'a> for Chart<'a> {
         self
     }
 }
+
+impl<'a> TickLabelBuilder<'a> for Chart<'a> {
+    /// Specifies the density of y-axis tick labels
+    fn y_tick_labels(&mut self, format: YAxisTickLabels) -> &mut Self {
+        match format {
+            YAxisTickLabels::None => {}
+            YAxisTickLabels::Sparse => {
+                self.height = if self.height < 16 {
+                    16
+                } else {
+                    self.height + 15 & !15
```

Comment:
I'd appreciate some comments about alignment math here

Reply:
also pls check lint errors below

Reply:
I agree that a name change sounds great. I'll update it today.

Reply:
Changed the math too be more readable using integer division instead to round off to the nearest multiple of 8 or 16. The original math added 7 (or 15) then rounded back down to the nearest 8 (or 16) by zeroing out the bottom 3 (or 4) bits with !&7 (or !&15)